locales/en/allocations.json

```diff
@@ -24,5 +24,36 @@
   },
   "allocation_criteria": {
     "page_title": "Allocation criteria"
+  },
+  "view": {
+    "page_title": "View allocation",
+    "heading": "{{count}} person from {{from_location}} to {{to_location}}, {{date}}",
+    "heading_plural": "{{count}} people from {{from_location}} to {{to_location}}, {{date}}",
+    "criteria_for_allocation": "Criteria for allocation",
+    "created_at": "Created {{datetime}}",
+    "fields": {
+      "estate_type": "Estate type",
+      "category": "Category",
+      "complex_cases": "Complex cases for prisons to agree",
+      "none_provided": "None provided",
+      "other_criteria": "Other criteria",
+      "sentence_length": {
+        "label": "Time left to serve"
```

Comment:
I think we still need to address the difference here. 

The label in the form journey is `Sentence length` but in the display we are saying `Time left to serve`. I think @Andy-Millar wanted to move more to this language, but either way we should be consistent with the label in the data capture and the replay of the data.

Reply:
It's not sentence length - we originally misunderstood and have updated to 'Time left to serve' for a good while now

Reply:
Sentence length is the name of the field described by the label. I think it's best to stick to that for the field names, as copy may change over time.

Reply:
No - because it's not sentence length - it's 'Time left to serve' (or sentence remaining). Sentence length and sentence remaining are completely different. Label and field needs to be 'Time left to serve' in all instances - allocation page and when PMU create an allocation

Reply:
I've marked this as unresolved as the label needs to be updated

Reply:
@barbaracassani I think you have misunderstood the issue. The issue is not with the `name` of the field, that's irrelevant and should match to the API for ease in sending that data.

My comments were around the mis-match in content we are displaying in two parts of the app. The label during creating a new allocation and when we display the answer to that information in the allocation detail should be the same. Which was my original suggestion to use the translation keys from the fields to ensure they will remain consistent. This is what we do in the move detail and something I think we should do here.

It's our responsibility to understand the codebase and the problem and to ensure consistency and if there are slight discrepancies that we see mock-ups we should have a quick conversation around them with @Andy-Millar and get them resolved.

The main goal is to create a consistent and simple to use UI rather than build exactly from screens. Sometimes content can be updated between tickets/changes so we just need to keep on eye on things to keep them inline.

Reply:
I have now updated the token `Sentence length` to read `Time left to serve`.